sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonSensor.java

```diff
@@ -110,6 +119,20 @@ public void execute(SensorContext context) {
     return Collections.unmodifiableList(list);
   }
 
+  private static PerformanceMeasure.Duration createPerformanceMeasureReport(SensorContext context) {
+    return PerformanceMeasure.reportBuilder()
+      .activate(context.config().getBoolean(PERFORMANCE_MEASURE_PROPERTY).orElse(Boolean.FALSE))
+      .allowSingleThreadMode()
```

Comment:
looks like this has been removed finally: https://github.com/SonarSource/sonar-analyzer-commons/pull/143/commits/fcf2dbd2009c5f68e85c4cdadc48b444bbedd716

Reply:
I updated the dependency accordingly. Anyway we'll still need to update it to its released version when available.